Car_owners/tests/test_views.py

```diff
@@ -212,6 +205,24 @@ def test_car_request_validation(
         assert response_get_car_invalid_data.status_code == status.HTTP_400_BAD_REQUEST
         assert response_get_car_invalid_data.data[key] == expected_message
 
+    @pytest.mark.django_db
+    def test_car_not_exist(
+        self,
+        api_client: APIClient,
+        valid_car_model_data: Car,
+        valid_new_car_view_data: dict[str, str | int],
+    ) -> None:
+        # car does not exist
+        car_id = valid_car_model_data.id
+        response_patch_car = api_client.patch(
+            f"/app/cars/{car_id + 1}/", data=valid_new_car_view_data, format="json"
+        )
+        response_delete_car = api_client.delete(
```

Comment:
let's keep it as is, so we don't keep changing this code so many times but patch and delete for a non existing object should also be separate tests. https://medium.com/javarevisited/how-solid-are-your-unit-tests-392b437ac310 Please remember this